static/src/javascripts/components/analytics/analytics.js

```diff
@@ -1,17 +1,48 @@
-@()
-@import conf.Static
-@import org.joda.time.DateTime
-
-try {
+//This uses the named module pattern : https://github.com/umdjs/umd/blob/master/templates/amdWebGlobal.js
```

Comment:
`components` is where bower components are extracted to (unclear I know), probably an unsafe place to keep this file